tests/test_control_basic_badcrc32/src/main.xc

```diff
@@ -218,19 +188,13 @@ int TestEp_Control(chanend c_out, chanend c_in, int epNum)
 int main()
```

Comment:
Quite a few main.xc files `#include "test_main.xc"` and use its common functions.  This file does not.  Is that intentional or an oversight?